client/src/components/EditProfile/EditForm/EditForm.tsx

```diff
@@ -59,7 +62,7 @@ const initialFormState = {
   address: '',
   phoneNumber: '',
   dateOfBirth: '',
-  availability: '',
+  availability: ['Monday'],
```

Comment:
Just realized I forgot to mention this:

We should be getting the initial state for this from the Profile context. Otherwise, everytime this page re-opened, we are going to display the same form no matter what the sitter availability is in the backend. This is true for the entire `initialFormState` so feel free to correct the other fields too.